(98) Style address selection

leeky opened the pull request:
The radio buttons for address selection are now rendered in the correct styling.

**NB: This branch is currently based off #5, as that branch includes the relevant CSS radio button styles from GOVUK elements.**

## Before
<img width="675" alt="before" src="https://user-images.githubusercontent.com/3166/30965903-ec425752-a44e-11e7-91c5-dac66a3ec0a9.png">

## After
<img width="318" alt="after" src="https://user-images.githubusercontent.com/3166/30965902-ec3e503a-a44e-11e7-9f08-6a356537711b.png">

leeky committed:
(chore) Use SimpleForm's 'input' to render radio buttons

Replace SimpleForm's `collection_radio_buttons` with the more high-level
`input` method which adds additional class names to the resulting HTML
markup, so that it can be styled correctly.

Additionally, add the 'button' class on the submit button.

NB: The GOVUK radio buttons don't work properly with
Capybara/Poltergeist tests because the styling hides the real form
control. Capybara can't click on hidden elements. Because of this, I've
changed the approach to click the associated label element instead.

To DRY this up, I created a capybara helper module which is included in
all feature specs.

dgmstuart commented:
rebased

dgmstuart commented on spec/support/capybara_helpers.rb:
```diff
@@ -0,0 +1,5 @@
+module CapybaraHelpers
+  def choose_radio_button(label)
```
Nice 👍

dgmstuart reviewed (changes requested):
As discussed, let's include the "Address Search" page styling in this task, and both these pages need a title heading

leeky committed:
(feature) Style the address search to look like the prototype

  - Added headings to the postcode search and address selection pages
  - Buttons get correct styles
  - Address selection radio buttons have correct label "Select your
    address"

leeky commented:
@dgmstuart As discussed, I've now styled the two forms so they now have the correct titles, buttons etc.

I've also rebased against develop.

dgmstuart commented:
Looks great 👍
